src/styles/levelSelect.scss

```diff
@@ -63,5 +65,30 @@
   display: inherit;
   grid-column: 3;
   justify-self: left;
-  padding-bottom: 2.5px;
+  margin-bottom: 13px;
+  margin-left: 5px;
+}
+
+.left-arrow {
+  bottom: 4.5px;
+  color: white;
+  left: 0;
+  margin-right: -1.5em;
+  position: absolute;
+  z-index: 1;
+}
+
+.right-arrow {
+  bottom: 4.5px;
+  color: white;
+  margin-left: -1.5em;
+  position: absolute;
+  right: 0;
+  z-index: 1;
+}
+
+.right-arrow:hover,
+.left-arrow:hover {
+  color: #adadad;
```

Comment:
idk how your LL is and whether you want to extrapolate your colors into the `global.scss`, I think it depends on whether this color is used in other parts.

<I also don't generally check in on the CSS>